.pre-commit-config.yaml

```diff
@@ -0,0 +1,60 @@
+---
+default_stages: [pre-commit]
+
+default_install_hook_types: [pre-commit, commit-msg]
+
+#ci:
```

Comment:
Maybe you shouldn't delete the entire section? Just `skip: [shellcheck]`. Also, you should delete it instead of commenting it out